internal/core/src/common/NullBitmapConcatenater.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#include <cstddef>
+#include <cstdint>
+#include <cstring>
+#include <utility>
+#include <vector>
+namespace milvus {
+
+class NullBitmapConcatenater {
+ public:
+    NullBitmapConcatenater() = default;
+
+    void
+    AddBitmap(uint8_t* bitmap, size_t size);
+
+    // Do not call this function multiple times
```

Comment:
Can we make `NullBitmapConcatenater` a function instead of a class, then?

Reply:
> Can we make `NullBitmapConcatenater` a function instead of a class, then?

it's ok if we define a function called NullBitmapConcatenater but I prefer use a class to manage allocated memory.